### PR TITLE
datos del receiver

### PR DESCRIPTION
**Pull request by Gustrucco**
[Finishes #114052519](https://www.pivotaltracker.com/story/show/114052519)

### CODE REVIEW

**Comment by canofm** [comment]
:+1:

**Comment by rodri042** [comment]
:+1:

File: lib/models/salesOrder.coffee [hunk 1]
```diff
@@ -65,4 +66,7 @@ class SalesOrder
     getTrackingNumber: =>
       if @shipments.length > 0 then "#" + @shipments[0].method.trackingNumber else ""
 
+    getReceiver: =>
+      if  @shipments.length > 0 then @shipments[0].receiver
```
Comment (NahuelP) [inline lib/models/salesOrder.coffee]: Que feo ver ese `if`x2 :disappointed:

**Comment by NahuelP** [comment]
:cocktail:

**Pull request merged**